framework/assets/yii.gridView.js

```diff
@@ -100,10 +108,12 @@
 
             $grid.find('form.gridview-filter-form').remove();
             var $form = $('<form action="' + url + '" method="get" class="gridview-filter-form" style="display:none" data-pjax></form>').appendTo($grid);
-            $.each(data, function (name, value) {
-                $form.append($('<input type="hidden" name="t" value="" />').attr('name', name).val(value));
+            $.each(data, function (name, values) {
+                $.each(values, function (index, value) {
+                    $form.append($('<input type="hidden" name="t" value="" />').attr('name', name).val(value));
```

Comment:
They will be sent, but PHP will use only the last one because `$_POST` is a key-value associative array.

When there are more than 1 selected option, you need to submit `name="login[]"` instead of `name="login"`.